README.md

```diff
@@ -335,6 +335,19 @@ The biggest difference between the the output of UMAP when compared with t-SNE i
 It generates a pseudo random number based on the seed and there are some famous algorithm, please see below link for further information on this.
 [[src]](https://en.wikipedia.org/wiki/Linear_congruential_generator)
 
+#### 64) Given that we want to evaluate the performance of 'n' different machine learning models on the same data, why would the following splitting mechanism be incorrect :
+```
+df = pd.DataFrame(...)
+rnd = np.random.rand(len(df))
+train = df[ rnd < 0.8 ]
+valid = df[ rnd >= 0.8 & rnd < 0.9 ]
+test = df[ rnd >= 0.9 ]
+```
```

Comment:
It would be more clear if you add the above code in a function and call it many times for different models so that the problem is more obvious. I can argue that you can use the same `train`, `valid` and `test` dataframes for different models without calling `rand()` again

Reply:
@andrewekhalel Thanks for your feedback ! I have integrated the changes you requested.